tests/bsim/bluetooth/audio/test_scripts/audio_config_tests/_cap_ac.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Copyright (c) 2025 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+dir_path=$(dirname "$0")
+
+set -e # Exit on error
+
+# Run all cap_ac_1* tests
+for file in "$dir_path"/_cap_ac_*.sh; do
```

Comment:
forgot this one?

Reply:
I purposely left those as is; they are calling the other ones that do `run_parallel`, and thought there was a minimum advantage of running these with `run_parallel`. If you think it's better to run script that call `run_parallel` with a script that runs `run_parallel`, let me know, and I'll update them. 

Alternatively these generic ones could also just call other the other non-`_` scripts with `run_parallel`

Reply:
I just thought you missed them.
Up to you, but could it as well just set the search path as `cap_ac*`? (instead of nesting calls)

Reply:
> Up to you, but could it as well just set the search path as cap_ac*? (instead of nesting calls) 

Yeah, that's the alternative I mentioned above :) 

I'd rather do that, than call the _cap_ac scripts in parallel :) Might be nice to have this, so will update